app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,47 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  # /foods
+  def index
+    @items = current_user.foods.order(created_at: :desc)
+  end
+
+  # /foods (post)
+  def create
+    food = Food.new(foods_params)
+    p food
```

Comment:
- [ ] <b>OPTIONAL:</b> Always remember to remove `logs` when you are done debugging .

Reply:
Thanks!